wwwroot/inc/snmp.php

```diff
@@ -2069,6 +2069,34 @@
 	'try_next_proc' => FALSE,
 );
 
+$iftable_processors['brocade-vdx-QSFP+'] = array
+(
+	'pattern' => '@^FortyGigabitEthernet 0/(\d+)$@',
+	'replacement' => 'Fo /0/\\1',
```

Comment:
So it accepts fo N/0/1 where N is the rbridge id.
As far as I can tell it doesn't expose that via SNMP.